tests/HtmlTest.php

```diff
@@ -1229,6 +1230,44 @@ public function testGetAttributeName(): void
         Html::getAttributeName('content body');
     }
 
+    public function wrapAttributeNameData(): array
+    {
+        return [
+            ['form', '[0]content', 'form[0][content]'],
```

Comment:
What does `[0]content` mean in an attribute name?

Reply:
Update name and description of this PR.